load-test/agoora-kafka-agent/docker-compose.yaml

```diff
@@ -0,0 +1,48 @@
+version: '3'
+services:
+  zookeeper:
+    image: confluentinc/cp-zookeeper:latest
+    environment:
+      ZOOKEEPER_CLIENT_PORT: 2181
+      ZOOKEEPER_TICK_TIME: 2000
```

Comment:
see cp-all-in-one there is one without zookeeper

Reply:
yeah I tried first with kraft but it included more config :D  I had one with zookeeper laying around. I will give it another go